server/templates/import_wizard.html

```diff
@@ -263,6 +263,8 @@
         <li>Confirm Mappings in the import wizard, one column at a time.
           <ul>
             <li>Identify Place, Variable, Date, and Observation Values Using the Tool</li>
```

Comment:
nit: "... using the Wizard" or drop that

Reply:
done